crates/swc_plugin_runner/Cargo.toml

```diff
@@ -50,5 +50,11 @@ swc_ecma_parser = { version = "0.102.0", path = "../swc_ecma_parser" }
 swc_ecma_visit = { version = "0.62.0", path = "../swc_ecma_visit" }
 testing = { version = "0.19.0", path = "../testing" }
 # This allows we can run non-wasm32 target build command while some pkg select features for wasmer/js omits its transitive deps
+criterion = "0.3"
```

Comment:
comment in here (https://github.com/swc-project/swc/pull/4397/files#diff-510e57dcbb462dd6fbc8c24bd5fc0317c4d9b625b1caa117cc10b2b3d5a30d21R52) is for the `wasmer`